src/pdf/DriverOverview/DriverOverviewPdf.tsx

```diff
@@ -155,7 +156,7 @@ const DriverOverviewCard: React.FC<DriverOverviewCardProps> = ({ data }) => {
                             <Text>{rowData.address.postcode}</Text>
                         </>
                     ) : (
-                        <Text>{nullPostcodeDisplay}</Text>
+                        <Text>{rowData.clientIsActive ? nullPostcodeDisplay : "-"}</Text>
```

Comment:
Why do we not use "-" for active clients who don't have a postcode?

Reply:
Active clients without a postcode can be assumed to be "NFA" (no fixed abode) clients, ie clients without an address. in this case, we want to display the postcode as "NFA". the value of nullPostcode is "NFA". Please see https://github.com/orgs/NorwoodAndBrixtonFoodbank/projects/1?pane=issue&itemId=57550936

Reply:
Ahh I see, I assumed it just means not applicable or something, maybe we should update the naming of that or add a tooltip 😄 unless this is a common abbreviation that only I don't know - can definitely see a volunteer get confused

Reply:
Cool, I've renamed `nullPostcodeDisplay` as I agree this was unclear!!

Reply:
If they are then all good, nothing else to do there